sqlglot/tokens.py

```diff
@@ -739,6 +739,7 @@ class Tokenizer(metaclass=_Tokenizer):
         "TRUNCATE": TokenType.COMMAND,
         "VACUUM": TokenType.COMMAND,
         "USER-DEFINED": TokenType.USERDEFINED,
+        "SYSDATE": TokenType.CURRENT_TIMESTAMP,
```

Comment:
delete this, and add it to doris

Reply:
I want to convert from other dialects to doris, only add the mapping in doris, it will not work for other dialects

Reply:
SYSDATE is not valid in every dialect, only certain dialects. if you want to map from CURRENT_TIMESTAMP to SYSDATE in doris, this is the the right approach.

Reply:
OK, the oracle dialect supports it, then I will write it in oracle